scripts/sheets_sync.py

```diff
@@ -181,13 +222,13 @@ def construct_update_dictionary(worksheet: list):
     # Loop through the form responses/data entry sheet and create a new row/list in the sheet_dict if a row/list doesn't
     # yet exist for the contact, or update the row/list if it already exists in sheet_dict
     for row in data:
-        email_address = row[signup_columns['Email Address']]
-        first_compilable_col = signup_columns['Zipcode'] + 1
+        email_address = row[signup_columns['email']]
```

Comment:
Since I added a birthyear column, that's the last column that remains independent. The rest of the columns are concatenated into one field. So throughout you'll see signup_columns['zipcode'] was converted to signup_columns['birtthyear']